browser/ui/views/frame/brave_browser_view.h

```diff
@@ -74,7 +74,7 @@ class BraveBrowserView : public BrowserView {
 #endif
   bool ShouldShowWindowTitle() const override;
   void OnThemeChanged() override;
-
+  bool HasSelectedURL() const override;
```

Comment:
Why this method is separated from other overridden methods below in private?
Caller doesn't need to call this method with `BraveBrowserView`.
We can call it via `browser()->window()->HasSelectedURL()`

Reply:
yep, done, thanks